src/main/java/org/apache/sling/servlets/post/impl/operations/ModifyOperation.java

```diff
@@ -70,31 +76,54 @@ public void setDateParser(final DateParser dateParser) {
 
     @Override
     protected void doRun(final SlingHttpServletRequest request,
-                    final PostResponse response,
-                    final List<Modification> changes)
-    throws PersistenceException {
+            final PostResponse response,
+            final List<Modification> changes)
+            throws PersistenceException {
         final Map<String, RequestProperty> reqProperties = collectContent(request, response);
 
         final VersioningConfiguration versioningConfiguration = getVersioningConfiguration(request);
 
         // do not change order unless you have a very good reason.
+        Instant start = Instant.now();
 
         // ensure root of new content
         processCreate(request.getResourceResolver(), reqProperties, response, changes, versioningConfiguration);
+        logger.debug("Performed create at {} path took {} ms to operate", request.getRequestURI(),
```

Comment:
I don't think that it is necessary to consistently add the value of ``request.getRequestURI()`` to the message. This information is always part of the context of the RequestProgressTracker, and also it won't change while execution it, it's static.
On top of that, ``Performed create at``  could be understood that a create is executed at this path, and this is not consistently true, as other resources could be affected, too (for example sub-resources).

I would just remove this part of the log.